Modules/_elementtree.c

```diff
@@ -4315,15 +4360,15 @@ static PyMethodDef _functions[] = {
     {NULL, NULL}
 };
 
-#define CREATE_TYPE(module, type, spec) \
-do {                                                                     \
-    if (type != NULL) {                                                  \
-        break;                                                           \
-    }                                                                    \
-    type = (PyTypeObject *)PyType_FromModuleAndSpec(module, spec, NULL); \
-    if (type == NULL) {                                                  \
-        goto error;                                                      \
-    }                                                                    \
+#define CREATE_TYPE(module, type, spec, base) \
```

Comment:
Is this new `base` parameter still needed? It seems to be always `NULL` in the uses below